blueocean-core-js/test/js/components/ReplayButton-spec.js

```diff
@@ -1,14 +1,14 @@
-/**
- * Created by cmeyers on 9/16/16.
- */
-
 import React from 'react';
 import { assert } from 'chai';
 import { shallow } from 'enzyme';
 
 import utils from '../../../src/js/utils';
 import { ReplayButton } from '../../../src/js/components/ReplayButton';
 
+
+jest.mock('../../../src/js/i18n/i18n');
```

Comment:
so this now has a mock i18n - whereas in the past it would just ignore it when running tests? (as not a new test)